sdk/cosmosdb/cosmos/package.json

```diff
@@ -94,6 +94,7 @@
     "@azure/core-auth": "^1.3.0",
     "@azure/core-rest-pipeline": "^1.2.0",
     "@azure/core-tracing": "^1.0.0",
+    "@azure/identity": "^2.0.1",
```

Comment:
core-auth is abstraction and identity is AAD implementation right? 

Why implementation dependency? Other libraries also don't have any such dependency.